dt-posts/dt-posts.php

```diff
@@ -179,6 +179,10 @@ public static function create_post( string $post_type, array $fields, bool $sile
                 $post_user_meta[$field_key] = $field_value;
                 unset( $fields[ $field_key ] );
             }
+            if ( $field_type === 'number' && $is_private ) {
+                $post_user_meta[$field_key] = $field_value;
+                unset( $fields[ $field_key ] );
+            }
```

Comment:
@micahmills would it be safe right here (and on contact update?) to add a catch all that would keep fields marked as private from being saved in the postmeta table?